packages/@react-aria/color/docs/useColorArea.mdx

```diff
@@ -216,35 +216,243 @@ function Example() {
-            height: 96,
-            marginRight: '2rem',
-            width: 96
+          ...trackProps.style
           }}></div>
-      </div>
-      <label>
-        {color.getChannelName(zChannel, 'en-US')}<br/>
-        <input
-          type="range"
-          min={color.getChannelRange(zChannel).minValue}
-          max={color.getChannelRange(zChannel).maxValue}
-          value={color.getChannelValue(zChannel)}
-          onChange={e => setColor(color.withChannelValue(zChannel, e.target.value))}
+        <div
+          className={`color-slider-thumb${isFocusVisible ? ' is-focused' : ''}`}
+          {...thumbProps}
           style={{
-            width: 192
-          }} />
-      </label>
-      <p>Current color value: {color.toString('hsb')}</p>
-    </>
+            ...thumbProps.style
+          }}>
+          <div className="color-slider-thumb-background"></div>
+          <div
+            className="color-slider-thumb-color"
+            style={{
+              background: state.getDisplayColor().toString('css')
+            }}></div>
+          <VisuallyHidden>
+            <input ref={inputRef} {...inputProps} {...focusProps} />
+          </VisuallyHidden>
+        </div>
+      </div>
+    </div>
   );
 }
 ```
+  <details>
+    <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show CSS</summary>
+```css
+.color-slider-track,
+.color-slider-track-background,
+.color-slider-track-color {
+  width: 100%;
+  border-radius: 4px;
+  forced-color-adjust: none;
+  position: relative;
+}
+
+.color-slider-track-background,
+.color-slider-track-color {
+  position: absolute;
+  height: 100%;
+}
+
+.color-slider-thumb {
+  position: absolute;
+  top: 14px;
+  border: 2px solid white;
+  box-shadow:
+    0 0 0 1px black,
+    inset 0 0 0 1px black;
+  border-radius: 50%;
+  box-sizing: border-box;
+  width: 20px;
+  height: 20px;
+}
+
+.color-slider-thumb.is-focused {
+  width: 32px;
+  height: 32px;
+}
+
+.color-slider-track-background,
+.color-slider-thumb-background {
+  background-size: 16px 16px;
+  background-position:
+    -2px -2px,
+    -2px 6px,
+    6px -10px,
+    -10px -2px;
+  background-color: white;
+  background-image:
+    linear-gradient(-45deg, transparent 75.5%, rgb(188, 188, 188) 75.5%),
+    linear-gradient(45deg, transparent 75.5%, rgb(188, 188, 188) 75.5%),
+    linear-gradient(-45deg, rgb(188, 188, 188) 25.5%, transparent 25.5%),
+    linear-gradient(45deg, rgb(188, 188, 188) 25.5%, transparent 25.5%);
+}
+
+.color-slider-thumb-background,
+.color-slider-thumb-color {
+  position: absolute;
+  border-radius: 50%;
+  width: 100%;
+  height: 100%;
+}
+```
+  </details>
+</details>
+
+### ColorSwatch
+
+The `ColorSwatch` component used in the example above implements an image preview of the color.
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
+
+```tsx example export=true render=false
+  function ColorSwatch (props) {
```

Comment:
We should probably pull this out into an actual React Spectrum/React Aria component at some point (not for this PR). Seems generally useful, and I think Spectrum has defined styles for it.

Reply:
I'm ok with that, but perhaps I'll leave it to React-Spectrum team to implement.